astria-rpc/build.rs

```diff
@@ -0,0 +1,9 @@
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+    let protos = vec!["proto/execution.proto"];
+
+    for proto in protos {
+        tonic_build::compile_protos(proto)?;
```

Comment:
`tonic_build` uses https://docs.rs/tonic-build/latest/tonic_build/struct.Builder.html#method.compile in their example, which basically does that heavy lifting for us.

Reply:
What heavy lifting are you referring to exactly?

Reply:
You pass it a list of protos and a list of includes, and it will compile all of them - basically what you are doing explicitly in the loop. See their example https://docs.rs/tonic-build/latest/tonic_build/index.html#examples